CHANGELOG.md

```diff
@@ -12,7 +12,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/).
   * Notifications when the channel a user subscribes to uploads new content ([#1066](https://github.com/lbryio/lbry-app/pull/1066))
   * Codacy support for Github contributions ([#1059](https://github.com/lbryio/lbry-app/pull/1059))
   * App category for Linux ([#877](https://github.com/lbryio/lbry-app/pull/877))
-  * Add YouTube Sync reward ([1147](https://github.com/lbryio/lbry-app/pull/1147))
+  * Add YouTube Sync reward ([#1147](https://github.com/lbryio/lbry-app/pull/1147))
```

Comment:
Good initiative!

Reply:
That was my bad :p I'll fix up the changelog before release for others.